fplus-lib/src/external_services/filecoin.rs

```diff
@@ -86,3 +87,20 @@ pub async fn get_allowance_for_client(address: &str) -> Result<String, reqwest::
         .await?;
     Ok(response.result)
 }
+
+pub async fn get_client_allocation(
```

Comment:
`external_services/filecoin.rs` and `models/filecoin.rs` contain stuff for communication with glif api.
datacapstats communication is in https://github.com/filecoin-project/filplus-backend/blob/main/fplus-lib/src/external_services/blockchain.rs

Also, blockchain.rs already has `get_verified_clients` function. What's different between that and the new one?